packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1140,6 +1140,23 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.deltaManager.on("op", this.onOp);
         }
 
+        // logging hardware telemetry
+        let deviceMemory;
+        let hardwareConcurrency;
+        if (typeof navigator !== "undefined") {
+            deviceMemory = Object.hasOwnProperty.call(navigator, "deviceMemory")
+            // eslint-disable-next-line @typescript-eslint/dot-notation
+                ? navigator["deviceMemory"] : undefined;
+            hardwareConcurrency = navigator.hardwareConcurrency;
+        } else {
```

Comment:
You really do not need this else block. You can just do 
let deviceMemory: string | undefined;
to keep compiler happy, as undefined is the default value.